aiida_quantumespresso/data/force_constants.py

```diff
@@ -5,6 +5,8 @@
 from qe_tools.constants import bohr_to_ang
 from aiida.orm import SinglefileData
 
+# pylint: disable=too-many-ancestors, raise-missing-from
```

Comment:
Prefer not to disable these here. The former I am disabling project wide in #562 so will soon no longer be necessary and the second would be better to actually fix